profile/README.md

```diff
@@ -1,3 +1,108 @@
+
+# How to Contribute
+Thank you for wanting to contribute to this project! We look forward to working with you. Here is a basic set of guidelines for contributing to our project. 
+* Please **checkout the issues** in the [deeper repository](https://github.com/the-deep/deeper) or the individual repository you want to contribute to. You can also create a new issue regarding what you want to work on. Please follow the guidelines mentioned above while creating the issue. You will need to include the link to the issue while creating a Pull Request(PR).
+* **Fork** the relevant repository.
+* **Make necessary changes** to the repository taking into account the **coding guidelines** mentioned in the individual repositories. Some general guidelines include the use of “git rebase” to organize commits and these instructions regarding commit messages:
+  * Separate subjects from body with a blank line
+  * Limit the subject line to 50 characters
+  * Capitalize the subject line
+  * Do not end the subject line with a period
+  * Use the imperative mood in the subject line
+  * Wrap the body at 72 characters
+  * Use the body to explain what and why vs. how
+* After your work is complete, **make a Pull Request** from your repository to the-deep repository. Describe what you did in as much detail as possible. Furthermore, please link to the issue in the description.
+* Our development team will go through the pull request and merge them if they are satisfactory. They can also review the PR and ask for explanations/modifications.
+
+# Repositories
+This project contains a number of repositories which work together. Here is a list of the main repositories and their brief descriptions:
+* **deeper**: The main repository for the DEEP project with configuration files and scripts for local and cloud deployments
+* **server**: Backend server with Django
+* **client**: Frontend code with React
+* **serverless**: Services for DEEP utilities that run in serverless framework
+* **deep-extensions**: Browser extension to add sources to DEEP platform
+
+# Installation Instructions
+* Clone deeper repo
+
+`git clone https://github.com/the-deep/deeper.git deep-project-root`
+* Go to deeper project root
+
+`cd deep-project-root`
+* Clone client, server and other repos
+
+`git clone https://github.com/the-deep/server.git server`
+
+`git clone https://github.com/the-deep/client.git client`
+
+`git clone --branch=feature/only-ary https://github.com/the-deep/client.git ./ary-only-client`
+
+`git clone https://github.com/the-deep/deepl-deep-integration.git deepl-service`
+
+* Comment out pulling of images from docker.io in docker-compose.yml if you want to build from the source
+
+`#image: docker.pkg.github.com/the-deep/client:latest`
+
+`#image: docker.pkg.github.com/the-deep/ary-only-client:latest`
+
+`#image: docker.pkg.github.com/the-deep/client:latest`
+
+`#image: docker.pkg.github.com/the-deep/deepl-service:latest`
```

Comment:
`docker-compose build` will build and tag the image locally using the tag specified in the docker-compose.yml. So we replace this with just
```suggestion
docker-compose build
```

Basically,
docker-compose pull # to pull pre-existing images if it exists. I think this is not maintained right now so maybe use build for now.
docker-compose build # to build from source.

Reply:
Makes sense. I have updated the file according to your suggestion. I didn't commit the exact suggestion because the docker-compose commands are listed below.